content/collections/data/en/amplitude-shopify-plugin.md

```diff
@@ -0,0 +1,48 @@
+---
+id: d7658703-c1c3-4cc4-a5e3-3b5674e41405
+blueprint: data
+title: 'Amplitude Shopify Plugin'
+landing: false
+exclude_from_sitemap: false
+updated_by: 0c3a318b-936a-4cbd-8fdf-771a90c297f0
+updated_at: 1729791421
+---
+The [Amplitude Shopify Plugin](https://apps.shopify.com/amplitude) enables you to instrument your Shopify store with an Amplitude features including [Autocapture](/docs/data/autocapture) and [Session Replay](/docs/session-replay).
+
+The Shopify Plugin plugin installs a version of the [Browser SDK](/docs/sdks/analytics/browser/browser-sdk-2), and adds the script before the `</head>` tag on each of your site's pages.
```

Comment:
Amplitude also sends all [Default Events](https://amplitude.com/docs/sdks/analytics/browser/migrate-from-marketing-analytics-to-browser-sdk-2-0#default-event-tracking).